client/src/client.ts

```diff
@@ -554,11 +554,15 @@ function extensionVersionCompatibleWithAllProjects(serverModuleLocation: string)
 /**
  * Returns true if any project in the workspace supports block syntax (v17+).
  */
-function getAngularVersionsInWorkspace(): NodeModule[] {
+async function getAngularVersionsInWorkspace(outputChannel: vscode.OutputChannel): Promise<NodeModule[]> {
+  const packageJsonFiles = await vscode.workspace.findFiles("**/package.json", "**/node_modules/**");
```

Comment:
Should we change the `node_modules` pattern to be something like `**/node_modules/@angular/core/**` to narrow down the amount of directories it needs to traverse?

Reply:
This is just the exclusion pattern. We'd want to exclude all `node_module` folders completely. I will add a comment.